Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -137,6 +137,15 @@ void TextureCacheBase::Invalidate()
   texture_pool.clear();
 }
 
+void TextureCacheBase::ForceReload()
+{
+  Invalidate();
+
+  // Force hi res textures to reload
+  HiresTexture::Update(true);
+  HiresTexture::Update(false);
```

Comment:
Why do you need to `Update` again after the "forced clear"?
If that is really needed this seems like an unintuitive interface.

Reply:
One update clears the hi resolution cache.  The second updates it with the new values.  I will add a new function to make things clearer (not sure why I did this).